src/jit-diff/jit-diff.cs

```diff
@@ -1243,6 +1245,7 @@ public int ListCommand()
             public string Arch {  get { return _arch;  } }
             public IReadOnlyList<string> AssemblyList => _assemblyList;
             public bool tsv {  get { return _tsv;  } }
+            public bool CCtors => _cctors;
```

Comment:
A micro-nit: you used a different capitalization for this bool in jit-dasm-pmi.cs